src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: transform $color-title-smoothly, color $color-title-smoothly;
```

Comment:
Use variable name for the transition duration instead of $color-title-smoothly.

Reply:
fix comments:
https://github.com/mate-academy/layout_catalog_hovers/pull/1937/files#r1185215760
https://github.com/mate-academy/layout_catalog_hovers/pull/1937/files#r1185215755